src/main/java/io/stargate/sgv2/jsonapi/service/processor/CommandProcessor.java

```diff
@@ -96,7 +96,10 @@ public <T extends Command, U extends SchemaObject> Uni<CommandResult> processCom
               }
               // But other exception types are unexpected, so log for now
               logger.warn(
-                  "Command '{}' failed with exception", command.getClass().getSimpleName(), t);
```

Comment:
This was wrong: `t` was taken as parameter for template, not as Throwable to print stack trace for (slf4j API oddity).